package.json

```diff
@@ -41,5 +42,8 @@
     "@jest/environment@npm:^27.5.1": "patch:@jest/environment@npm%3A28.1.3#~/.yarn/patches/@jest-environment-npm-28.1.3-506a81a227.patch",
     "@jest/environment@npm:^28.1.3": "patch:@jest/environment@npm%3A28.1.3#~/.yarn/patches/@jest-environment-npm-28.1.3-506a81a227.patch",
     "superstruct": "patch:superstruct@npm%3A1.0.4#~/.yarn/patches/superstruct-npm-1.0.4-44d328b887.patch"
+  },
+  "dependencies": {
+    "stack-utils": "^2.0.6"
```

Comment:
I don't think we need to (or should?) add this tot he root `package.json`. We probably _should_ add it to the one in `packages/replay` though (even if we don't plan any more releases of this project)

Reply:
Okay